packages/cli/tests/unit/fulltext/querySymbols.spec.ts

```diff
@@ -8,27 +8,60 @@ describe('querySymbols', () => {
       const symbols = querySymbols(srcPath);
       expect(symbols.sort()).toStrictEqual(
         [
-          'ClassOne',
```

Comment:
It's very important for the index that CamelizedWords and snake_case be left as-is. Because it's relying on that structure to be able to split words apart.